pyphare/pyphare_tests/pharein/simulation_test.py

```
@@ -77,6 +77,119 @@ def test_time_step(self):
+                                "B": 0.5,
+                            },
+                        },
+                        "xupper": {"type": "super-magnetofast-outflow"},
+                    },
+                )
+            )
+
+    def test_callable_B_only_for_super_magnetofast_inflow(self):
+        # a time-function B is accepted for super-magnetofast-inflow but not free-pressure-inflow
+        global_vars.sim = None
+        Bt = lambda t: [0.5, 1.0, 0.0]
```

Ruff (0.15.20)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `Bt` as a `def`

(E731)